platformio.ini

```diff
@@ -21,6 +21,6 @@ board_build.f_cpu = 27000000L
 ; extra_scripts = post_extra_script.py
 
 upload_protocol = stcgal
-upload_port = /dev/ttyUSB0
+upload_port = COM4
```

Comment:
We need to figure out how to specify the upload_port in a way that will not trigger this line to be modified. People will have different ports and they need the ability to override this setting without modifying the code.